sources/osgShadow/ShadowReceiveAttribute.js

```diff
@@ -167,6 +165,7 @@ utils.createPrototypeStateAttribute(
 
             uniformMap.normalBias.setFloat(this._normalBias);
             uniformMap.bias.setFloat(this._bias);
+            uniformMap.pcfKernel.setFloat(this._pcfKernel);
```

Comment:
Why not using int ?

Reply:
float allow for 'fuzzy' pcf values like values computed from an algo if need be. (distance to blocker, size of light)  

As it's a uniform and int and float same size, it has no incidence whatsoever and yet more flexible